FirebaseFunctions/Tests/Unit/FunctionsTests.swift

```diff
@@ -358,4 +358,73 @@ class FunctionsTests: XCTestCase {
     }
     waitForExpectations(timeout: 1.5)
   }
+
+  func testGenerateStreamContent() async throws {
+    let options = HTTPSCallableOptions(requireLimitedUseAppCheckTokens: true)
+
+    let input: [String: Any] = ["data": "Why is the sky blue"]
+    let stream = try await functions!.stream(
+      at: URL(string: "http://127.0.0.1:5001/demo-project/us-central1/genStream")!,
+      withObject: input,
+      options: options,
+      timeout: 4.0
+    )
+    let result = try await response(from: stream)
+    XCTAssertEqual(
+      result,
+      [
+        "chunk hello",
+        "chunk world",
+        "chunk this",
+        "chunk is",
+        "chunk cool",
+        "hello world this is cool",
```

Comment:
I suspect this final concatenation was a result of GTMSessionFetcher handling the streamed results. I don't expect it with URLSession and AsyncBytes